app/metrics.yaml

```diff
@@ -5809,3 +5809,156 @@ recent_tabs:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-06-23"
+credit_cards:
+  saved:
+    type: counter
+    description: |
+      A counter of the number of credit cards that have been saved
+      manually by the user.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/18711
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/20909
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-09-01"
+  deleted:
+    type: counter
+    description: |
+      A counter of the number of credit cards that have been deleted by
+      the user.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/18711
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/20909
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-09-01"
+  modified:
+    type: event
+    description: |
+      A credit card has been modified by the user.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/18711
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/20909
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-09-01"
+  form_detected:
+    type: event
+    description: |
+      A credit card form was detected.
+    send_in_pings:
+      - metrics
```

Comment:
Is this intentional? Events are usually sent in the [events](https://mozilla.github.io/glean/book/user/pings/events.html) ping. Note that you can also leave this property out for Glean to send the data automatically in the correct ping.